pkg/devfile/adapters/kubernetes/adapter.go

```diff
@@ -37,7 +37,39 @@ func (k Adapter) Push(parameters common.PushParameters) error {
 	return nil
 }
 
-// DoesComponentExist returns true if a component with the specified name exists
+// Build creates Kubernetes resources to build an image for the component
+func (k Adapter) Build(parameters common.BuildParameters) error {
```

Comment:
Is there any reason we use `k` for the Kubernetes Adapter and `a` for the docker one? Throughout the rest of the codebase I think Adapters are `a` (even the Kube ones) - I don't know if that's worth changing, though. Just a suggestion (more of a nitpick) for standardisation.

Reply:
Yeah good point!

Reply:
I think we just copied what was there originally for the Push command.